jobs/update_hathifile_listing.rb

```diff
@@ -49,6 +50,7 @@ def run
 
     file_list_file = File.open(hathifile_listing, "w")
     file_list_file.puts file_list.sort_by { |hf| hf["filename"] }.to_json
+    tracker.log_final_line
```

Comment:
No tracker.increment in this script? Can't tell if omitted intentionally or un-.

Reply:
That's probably an omission, good catch - the test doesn't check for a nonzero number.  We could probably increment for each file in the `hathifile_web_dir` that we examine.

Reply:
Added.